### DISCUSSION TITLE
Place for GCOV kernel discussion

### DISCUSSION THREAD

**Discussion by joefradley** [linux-test-project/lcov#198] [Q&A] [answered]
I'm wondering if this project could also be the place for general GCOV kernel questions. If not here, where would be the best place?

**Comment by henry2cox** [comment, marked as answer]
No idea if anybody follows lcov discussions (we only just set that up)...and really no idea whether anyone could answer your questions – but seems painless to try.
Worst case:  wrong place/no answers/no help – and then we/you need to find a better place.

**Reply by joefradley** [reply]
I feel like there's a good chance. I was excited when I saw @oberpar comment on the other discussion. He's the original author! :) . I'll post my question as a separate thread. Thank you.
